spec/models/customer_spec.rb

```diff
@@ -6,4 +6,44 @@
     it {should have_many(:invoices)}
 
   end
+
+  describe 'class methods' do
+    it 'shows top customers' do
+      customer1 = Customer.create!(first_name: 'Gunther', last_name: 'Guyman')
+      customer2 = Customer.create!(first_name: 'Miles', last_name: 'Prower')
+      customer3 = Customer.create!(first_name: 'Mario', last_name: 'Mario')
+      customer4 = Customer.create!(first_name: 'Marneus', last_name: 'Calgar')
+      customer5 = Customer.create!(first_name: 'Sol', last_name: 'Badguy')
+      customer6 = Customer.create!(first_name: 'Wyatt', last_name: 'Kribs')
+
+      invoice1 = Invoice.create!(customer_id: customer1.id, status: 2)
+      invoice2 = Invoice.create!(customer_id: customer2.id, status: 2)
+      invoice3 = Invoice.create!(customer_id: customer3.id, status: 2)
+      invoice4 = Invoice.create!(customer_id: customer4.id, status: 2)
+      invoice5 = Invoice.create!(customer_id: customer5.id, status: 2)
+
+      transaction1 = Transaction.create!(invoice_id: invoice1.id, credit_card_number: 12345, result: 1)
+      transaction2 = Transaction.create!(invoice_id: invoice2.id, credit_card_number: 12345, result: 1)
+      transaction3 = Transaction.create!(invoice_id: invoice3.id, credit_card_number: 12345, result: 1)
+      transaction4 = Transaction.create!(invoice_id: invoice4.id, credit_card_number: 12345, result: 1)
+      transaction5 = Transaction.create!(invoice_id: invoice5.id, credit_card_number: 12345, result: 1)
+
+      expect(Customer.top_customers).to eq([customer1, customer2, customer3, customer4, customer5])
```

Comment:
This test doesn't really test your functionality of the .top_customers method. Since each customer only has one transaction and the method is supposed to return the top five customers by successful transactions, it's just going to return the customers in the order they were created because they each have only one customers. I would create at least 15 transactions so you can have one customer have 5 transactions, one have 4,  one have 3... etc.